src/cli.ts

```diff
@@ -27,14 +32,37 @@ async function main(argv: minimist.ParsedArgs) {
   const argOut: string = argv._[1] || argv.output
 
   try {
-    const schema: JSONSchema4 = JSON.parse(await readInput(argIn))
-    const ts = await compile(schema, argIn, argv as Partial<Options>)
-    await writeOutput(ts, argOut)
+    if (isGlob(argIn)) {
+      if (!existsSync(argOut)) {
+        await mkdir(argOut)
+      }
+
+      const files = await glob(join(process.cwd(), argIn))
```

Comment:
Could we move this up (above creating the output directory), and short-circuit early with a nice error message when `files.length === 0` (something like "Couldn't find any files using the glob [glob].")?